python/tests/aggregate_tests/atest_register_tests.py

```diff
@@ -20,17 +17,3 @@ def register_tests_in_module(module, ta: TstAccumulator, class_name: str):
                 instance.register(ta)
                 if DEBUG:
                     print(f"Registering {name}")
-
```

Comment:
I would move the function run from atest_run.py back here, after renaming it `discover_tests` (see the comment there). For discovering tests, this file is a better match.